eva_sub_cli/validators/docker_validator.py

```diff
@@ -12,7 +12,7 @@
 logger = logging_config.get_logger(__name__)
 
 container_image = 'ebivariation/eva-sub-cli'
-container_tag = 'v0.0.1.dev10'
+container_tag = 'v0.0.1.dev11'
```

Comment:
`dev11` does not exist in dockerhub which is why the test are failing

Reply:
published the new version in dockerhub